clang/lib/Sema/SemaDecl.cpp

```diff
@@ -20008,6 +20008,90 @@ bool Sema::IsValueInFlagEnum(const EnumDecl *ED, const llvm::APInt &Val,
   return !(FlagMask & Val) || (AllowMask && !(FlagMask & ~Val));
 }
 
+bool Sema::ComputeBestEnumProperties(ASTContext &Context, bool isPacked,
+                                     unsigned NumNegativeBits,
+                                     unsigned NumPositiveBits,
+                                     QualType &BestType,
+                                     QualType &BestPromotionType) {
+  unsigned IntWidth = Context.getTargetInfo().getIntWidth();
```

Comment:
Also, if we’re moving this into `ASTContext`, we should do a small cleanup pass while we’re at it and replace `Context.getTargetInfo().` w/ `Target->` in this function since we can just access it directly at that point.